eatin-client/src/pages/RecipePage/RecipePage.tsx

```diff
@@ -63,6 +77,12 @@ export const RecipePage: FC = () => {
         }
     }
 
+    const setNewRating = (newValue: number | null) => {
```

Comment:
check if this function is called several times, I think it gets called when the component renders with undefined values

Reply:
its called once when you give a recipe a new rating